### ISSUE TITLE
默认不要翻译剪贴板里的内容

### ISSUE THREAD

**Issue by xtuer** [raycast/extensions#9559]
### Extension

https://www.raycast.com/xeric/simple-youdao

### Description

打开翻译界面后，默认会译剪贴板里的内容，对于我来说这个操作是不需要的，可以在配置里增加一项，设置是否默认翻译剪贴板里的内容吗？

**Comment by xtuer** [comment]
是插件的问题，不是 raycast 的。

**Comment by xeric** [comment]
@xtuer 
Good suggest, thanks! Will add ASAP.
Requirement: add a option for this extension to disable/enable translate the content in clipboard.

**Comment by xtuer** [comment]
> @xtuer Is this what you want? ![CleanShot 2023-12-12 at 07 52 13](https://private-user-images.githubusercontent.com/190927/289703053-b1328ca7-e694-4e40-bcbd-e286c5514063.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNTIzMjEsIm5iZiI6MTcwMjM1MjAyMSwicGF0aCI6Ii8xOTA5MjcvMjg5NzAzMDUzLWIxMzI4Y2E3LWU2OTQtNGU0MC1iY2JkLWUyODZjNTUxNDA2My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMlQwMzMzNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMTljM2M1MGI1YTc3Yzg5MzVlN2RlZDFjOGM0ZDViYTNhNmM4NzVhMGZmNWY4ZDZkYTBjYjQwNmI0YmYxYmI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.F3BVObKVTF_aG81-jDBt9_3erzdCWEKX3UudrvAhRkM)

Same as your reply.
Add options to disable translate content in clipboard, or only translate the user manually input content.

**Comment by xeric** [comment]
@xtuer if you uncheck this checkbox, then you will not get any translation from your clipboard content.

**Comment by xtuer** [comment]
> @xtuer if you uncheck this checkbox, then you will not get any translation from your clipboard content.

Thank you very much!

My request is about extension "Simple Youdao Translate".

**Comment by xeric** [comment]
> > @xtuer if you uncheck this checkbox, then you will not get any translation from your clipboard content.
> 
> Thank you very much!
> 
> My request is about extension "Simple Youdao Translate".

Yes, it is, I'm the author of this extension, and above screenshot is from this extension.
![CleanShot 2023-12-15 at 11 24 18](https://github.com/raycast/extensions/assets/190927/f8a60706-1b0a-4475-920b-841d448be3c4)

**Comment by xtuer** [comment]
> > > @xtuer if you uncheck this checkbox, then you will not get any translation from your clipboard content.
> > 
> > 
> > Thank you very much!
> > My request is about extension "Simple Youdao Translate".
> 
> Yes, it is, I'm the author of this extension, and above screenshot is from this extension. ![CleanShot 2023-12-15 at 11 24 18](https://private-user-images.githubusercontent.com/190927/290716451-f8a60706-1b0a-4475-920b-841d448be3c4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI2MTk1MTUsIm5iZiI6MTcwMjYxOTIxNSwicGF0aCI6Ii8xOTA5MjcvMjkwNzE2NDUxLWY4YTYwNzA2LTFiMGEtNDQ3NS05MjBiLTg0MWQ0NDhiZTNjNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxNVQwNTQ2NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NzljZWM0MTg1NjM0MTczNTdmYTc1OTk1OTdmZmZhOWJiM2VjODBmODhiMjc5N2U1NjMyY2Q5NWE1OGFlOTJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.teYlwrlLUoMVU0umwtej5QlmhY6ys6IZwWynlb4xe-c)

Thank you, it works fine.
